cmd/krel/cmd/changelog.go

```diff
@@ -313,7 +314,8 @@ func (c *Changelog) run(opts *changelogOptions, rootOpts *rootOptions) error {
 	}
 
 	logrus.Info("Generating TOC")
-	toc, err := notes.GenerateTOC(markdown)
+	toc, err := mdtoc.GenerateTOC([]byte(markdown))
+	fmt.Println(toc)
```

Comment:
Can we remove this line?

Reply:
Yep, sorry. It was to debug and forgot to remove it. DOne.